src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -619,10 +619,8 @@
           "Percentage (0-100) of the old gen allowed as dead wood. "        \
           "Serial mark sweep treats this as both the minimum and maximum "  \
           "value. "                                                         \
-          "Par compact uses a variable scale based on the density of the "  \
-          "generation and treats this as the maximum value when the heap "  \
-          "is either completely full or completely empty.  Par compact "    \
-          "also has a smaller default value; see arguments.cpp. "           \
+          "Parallel full gc treats this as maximum value, i.e. a non-fully" \
+          "compact full gc cycle wastes at most this value of space. "      \
```

Comment:
```suggestion
          "Parallel full gc treats this as maximum value, i.e. when "       \
          "allowing dead wood, Parallel full gc wastes at most this amount "\
          "of space."                                                       \
```
I admit I did not fully understand the original description, so this is my best guess what had been meant. First, "non-fully compact full gc cycle" sounds very awkward as "non-fully" isn't defined anywhere, and what is a full gc "cycle" wrt Parallel GC?  Also s/value/amount.

Maybe also change the term "mark sweep" for Serial to "full gc" above as well because it seems to be unnecessary to refer to the algorithm here.

Reply:
I meant non-"MaximumCompaction".